mouse_test.go

```diff
@@ -54,9 +54,14 @@ func TestBasicMouseMoves(t *testing.T) {
 		t.Fatalf("Failed to perform right click. Last error was: %s\n", err)
 	}
 
+	err = relDev.MiddleClick()
+	if err != nil {
+		t.Fatalf("Failed to perform middle click. Last error was: %s\n", err)
+	}
+
 	err = relDev.LeftPress()
 	if err != nil {
-		t.Fatalf("Failed to perform left key press. Last error was: %s\n", err)
+		t.Fatalf("Failed to perform left key isPress. Last error was: %s\n", err)
```

Comment:
Quick question — why the change from `press` to `isPress` here and elsewhere?

Reply:
This must also have happened during some refactoring, I have changed it back. Sorry, I should have checked the diff more carefully before creating a pull request :)

Reply:
Haha. Thanks!